src/collections/_documentation/workflow/releases/release-automation/bitbucket-pipelines/index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Bitbucket Pipelines
+sidebar_order: 2
+
+---
+
+[Bitbucket Pipelines: Sentry Release Pipe](https://bitbucket.org/sentryio/sentry-new-release)
```

Comment:
Do we need this line? Don't feel like it's adding much cause we have a title above and the link below.

Reply:
I just wanted to have a link front and center to where they can find the integration. @MimiDumpling thoughts?

Reply:
Hi! Since the link is already present in the paragraph on line 9, I'd remove the link on line 7. ;) Thanks for updating the docs!